playbooks/engineer-workflow.md

```diff
@@ -91,9 +91,13 @@ Reviewers are asked to leave their review within one business day of being reque
 any reason, it's the reviewer's responsibility to communicate with the author of the PR to establish a different
 timeline or get someone else to review.
 
-**_Assign_** another team member (and only _one_ team member) to the PR. This is the person who is responsible for
-moving the PR forward. Once someone has been assigned to a PR, they should make a good faith effort to take one of
-the following actions within one business day:
+**_Assign_** a single team member (they will often also be one of your reviewers) to the PR. This is the person who
+is responsible for moving the PR forward.
+
+Assigning exactly one reviewer - no more and no less - helps ensure that there's no question about who is
+responsible. Since assignees are not explicitly expected to review, it's perfectly fine to both assign and request
+a review from the same person. Once someone has been assigned to a PR, they should make a good faith effort to take
+one of the following actions within one business day:
```

Comment:
If the `Assignee` is responsible for moving the PR forward, then it should be expected that they're responsible for line-by-line reviewing. 

With the `Assignee` being responsible for moving a PR forward, any number of `Reviewers` can then be assigned as additional sets of eyes. Often times a PR covers a lot of domain territory that involves different expertise. Why would be want to explicitly define how feedback is received in this way?

@joeyAghion noted earlier that often times _too_ many reviewers can lead to conflicting requests for changes. However, in practice, it's rare that this happens as typical PRs are dev -> assignee, without additional reviewers. When additional reviewers are requested, this is typically an action taken by the developer who is specifically looking for additional feedback. This shouldn't be discouraged.

Reply:
@damassi I'm not sure I follow - I don't see this as limiting the amount of reviewer feedback or discouraging additional review requests. Help me understand where you're coming from?

Reply:
This bit of language: 

> Assigning exactly one reviewer - no more and no less - helps ensure that there's no question about who is
responsible

As well as this: 

> Since assignees are not explicitly expected to review

Reply:
Ah, reading through the context I see this is clarified by sub-headings, all good 👍